tests/unittests/utils/test_ultralytics.py

```
@@ -0,0 +1,836 @@
+        assert cls.__name__ == "YOLOEVPSegPredictor"
+
+    def test_raises_import_error_with_attribute_error_cause(self, monkeypatch):
+        from fiftyone.utils import ultralytics as fu
+
+        class _BrokenYoloe:
+            def __getattr__(self, name):
+                raise AttributeError(name)
+
+        monkeypatch.setattr(fu, "_yoloe", _BrokenYoloe())
+
+        with pytest.raises(ImportError, match="ultralytics>=8.4.0") as exc:
```

Ruff (0.15.12)

[warning] Pattern passed to `match=` contains metacharacters but is neither escaped nor raw

(RUF043)